README.md

```diff
@@ -1,75 +1,328 @@
+Conditions:          
+ Completed           True
+Created:             Wed Feb 03 09:34:22 -0500 (30 minutes ago)
+Started:             Wed Feb 03 09:34:22 -0500 (30 minutes ago)
+Finished:            Wed Feb 03 09:46:51 -0500 (17 minutes ago)
+Duration:            12 minutes 29 seconds
+Progress:            
+ResourcesDuration:   22m11s*(1 cpu),22m11s*(100Mi memory)
 
-With the provided anonymizer script, you can export a copy of your production cluster schema and replicate it in the
-Kubernetes environment for testing.
+STEP                           TEMPLATE                                            PODNAME                        DURATION  MESSAGE
+ ✔ workflow-adelphi-1          execute                                                                                                                    
+ ├─✔ cassandra-source          cassandra-clusters/start-source-cluster             workflow-adelphi-1-1714058278  30s                                     
+ ├─✔ start-registry            registry/registry-template                          workflow-adelphi-1-543986670   29s                                     
+ ├─○ build-cassandra           cassandra-image/cassandra-image-template                                                     when 'false' evaluated false  
+ ├─○ build-cassandra-mgmt-api  cassandra-mgmt-image/cassandra-mgmt-image-template                                           when 'false' evaluated false  
+ ├─✔ cassandra-target          cassandra-clusters/start-target-cluster             workflow-adelphi-1-1333841590  2s                                      
+ ├─✔ cassandra-ready           cassandra-status/ready                              workflow-adelphi-1-1071284282  3m                                      
+ ├─✔ configure-schema          schema-job/configure-schema                         workflow-adelphi-1-462681126   11s                                     
+ ├─✔ nosqlbench-source         nosqlbench-job/nosqlbench-source                    workflow-adelphi-1-1304337547  1m                                      
+ ├─✔ nosqlbench-target         nosqlbench-job/nosqlbench-target                    workflow-adelphi-1-2207475351  1m                                      
+ ├─✔ run-diff                  spark-job/spark-job-template                        workflow-adelphi-1-2458661793  4m                                      
+ ├─✔ collect-diff-results      cassandra-diff/collect-results                      workflow-adelphi-1-1334654991  16s                                     
+ ├─✔ gemini-oracle-sut         gemini-job/gemini-oracle-sut                        workflow-adelphi-1-3173132751  2m                                      
+ └─✔ start-results-server      results/results-server                              workflow-adelphi-1-1122498803  2s  
+ ```
 
-## Pre-requisites
+> Note that the `build-cassandra` and `build-cassandra-mgmt-api` steps of the workflow may not be executed with all runs of the workflow.
```

Comment:
since we're mentioning it, it's probably good to let people know why: `unless you have specified a git_identifier to build C* from the source` (or something along those lines).

Reply:
👍